frontend/src/components/AddWordModal.tsx

```diff
@@ -0,0 +1,136 @@
+import { useEffect, useState } from 'react';
+import { Button, Form, Modal, Row, Col } from 'react-bootstrap'
+import Choice from '../interfaces/choice';
+import Question from '../interfaces/question';
+
+interface Props {
+  show: boolean,
+  categoryId: number,
+  handleClose: () => void,
+  handleCreateQuestionWithChoices: (word: Question, choice: Choice[]) => void
+}
+
+function AddWordModal(props: Props) {
+  const {
+    show,
+    categoryId,
+    handleClose,
+    handleCreateQuestionWithChoices,
+  } = props;
+
+  const [word, setWord] = useState<Question>({ value: "" });
+  const [choices, setChoices] = useState<Choice[]>([
+    { value: "", is_correct_answer: false },
+    { value: "", is_correct_answer: false },
+    { value: "", is_correct_answer: false },
+    { value: "", is_correct_answer: false },
```

Comment:
can this be dynamic? Depending on what is returned from BE

Reply:
gituyo nako ni sir para ani:
![image](https://user-images.githubusercontent.com/57890826/193181747-8a2da724-c85f-45c5-90fa-c205bcb675c5.png)

or unsay better way to implement this sir? hehe

Reply:
@wylenjoan Hmmm its kinda static. Akong concern lang is what if mag add ug Choice? Mag add sad ka ug extra object sa state? Maka add ba ug choice? or 4 ra kutob?

Reply:
@ziontavera Wala ko nakathink nga pwede mag-add ug choice sir hahaha. Maybe static lang sah for now unya will improve it if naay extra time 😅